spec/faraday/adapter/em_synchrony_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+RSpec.describe Faraday::Adapter::EMSynchrony do
+  features :request_body_on_query_methods, :reason_phrase_parse, :skip_response_body_on_head, :parallel
+
+  it_behaves_like 'an adapter'
+
+  it 'allows to provide adapter specific configs' do
+    url = URI('https://example.com:1234')
+    adapter = Faraday::Adapter::EMSynchrony.new nil, inactivity_timeout: 20
+    req = adapter.create_request(url: url, request: {})
+
+    expect(req.connopts.inactivity_timeout).to eq(20)
```

Comment:
Q: Is `#connopts` shared by all kinds of adapters, or only by EmSynchrony? That is: Do we have an oppportunity for more shared test setup code?

(This is me being curious, not a blocker or a criticism.)

Reply:
`create_request` returns an `EventMachine::HttpRequest` so it's only for `em-http` and `em-synchrony`